### DISCUSSION TITLE
How can you select the "view full transaction details" button when approving token spend?

### DISCUSSION THREAD

**Discussion by 0xLienid** [Synthetixio/synpress#469] [Q&A] [answered]
There doesn't seem to be a function built in to click the "View full transaction details" button to display everything from the Metamask approval request notification and attempting to use cy.switchToMetamaskNotification() to then manually select it throws an error `Converting circular structure to JSON
    --> starting at object with constructor 'BrowserContext'
    |     property '_browser' -> object with constructor 'Browser'
    --- property '_defaultContext' closes the circle`
    
    Any ideas?

**Comment by drptbl** [comment, marked as answer]
There is no such functionality in synpress yet, it has to be added.

Thanks for suggestion,
Jakub.
